docs/date.md

```diff
@@ -1,96 +1,398 @@
 # Temporal.Date
 
-A representation of a calendar date.
+A `Temporal.Date` represents a calendar date.
+"Calendar date" refers to the concept of a date as expressed in everyday usage, independent of any time zone.
+For example, it could be used to represent an event on a calendar which happens during the whole day no matter which time zone it's happening in.
+
+`Temporal.Date` refers to the whole of a specific day; if you need to refer to a specific time on that day, use `Temporal.DateTime`.
+A `Temporal.Date` can be converted into a `Temporal.DateTime` by combining it with a `Temporal.Time` using the `withTime()` method.
+
+`Temporal.YearMonth` and `Temporal.MonthDay` carry less information than `Temporal.Date` and should be used when complete information is not required.
 
 ## Constructor
 
 ### **new Temporal.Date**(_year_: number, _month_: number, _day_: number, _disambiguation_: 'constrain' | 'balance' | 'reject' = 'constrain') : Temporal.Date
 
-Creates a new `Date` object that represents a calendar date
+**Parameters:**
+- `year` (number): A year, ranging between &minus;271821 and 275760 inclusive.
+- `month` (number): A month, ranging between 1 and 12 inclusive.
+- `day` (number): A day of the month, ranging between 1 and 31 inclusive.
```

Comment:
Please add calendar as an argument.

Reply:
I assume it's OK to do this later along with the rest?

Reply:
ok